tests/security/stig/baseline_comparison.pm

```diff
@@ -0,0 +1,51 @@
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-Later
+#
+# Summary: Compare the baseline before and after remediation, generate a log
+# Maintainer: shawnhao <weixuan.hao@suse.com>
```

Comment:
I think you forgot to put poo here  #[104944]

Reply:
poo tag added